src/Sinch/SMS/SmsServicePlanIdHostingRegion.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Sinch.SMS
```

Comment:
Not sure this will be an added value from SDK user's perspective : defining a dedicated type for supported region in case of service plan id usage
This mean if for test or any other reason user switch from `projectId` <-> `servicePlanId` usage the region as also to be changed. Not because wanting to changed to the region but because type is different.
e.g.: Still addressing `us` region in both cases but have to change `SmsHostingRegion.Us` <-> `SmsServicePlanIdHostingRegion.Us`
Even if supported regions are not equals, a dynamic validation against unexpected values should be enough.
And even in this case: it should be a warning and not an error. So in case of server going forward to support new region, SDK's user do not need to move to a new (but not yet ready release) SDK updated version

Reply:
I'll keep dedicated type for regions as sets of supported are different. For any 'I don't want checks' users it's a record so `new SmsServicePlainIdHostingRegion('cn')` should work.

Reply:
From the SDK user's point of view, a region is just a region: the EU, the USA, ... are no different targets when approached by service plan or project.
What is the added value of complexity for SDK's user?

Reply:
The supported set of regions  _are_ different, and different enums reflects just that.

Reply:
This has to be discussed within the team. Item added in the parking lot